README.md

```diff
@@ -1,2 +1,19 @@
-# media
-Drupal 8 version of the Drupal media module.
+# Media Module
+
+A complete media module for Drupal 8.
+
+## Requirements
+
+* Latest dev release of Drupal 8.x.
```

Comment:
We also already know we'll depend on media_entity & friends. We'll also add them as we progress...